momentum/4/modules/ipv-6-lookup.md

```diff
@@ -3,17 +3,17 @@ title: "ipv6_lookup – Multi-address-family MX Records"
 description: "Configuration Change This feature is available in Momentum 4 2 and later The ipv 6 lookup module supports multi address family MX records enabling A record lookups for I Pv 6 addresses This module keeps track of whether the A record query succeeded and if it did when getting the..."
 ---
 
-<a name="idp21948288"></a> 
+<a name="idp21948288"></a>
 
-**Configuration Change. ** This feature is available in Momentum 4.2 and later.
+**Configuration Change. ** This feature only needs to be enabled on 4.2.x versions. 4.3.0 and higher already has it built in.
```

Comment:
I think the wording here could use a little improvement. 
Does this mean that the module only needs to be enabled in the configuration on 4.2.x, and is automatically enabled in 4.3.0 and higher? Was the feature made available in 4.2, but 4.2.x needs the module to be loaded and 4.3.0 does not need to load the module? I think removing when the feature was made available could confuse some customers. Maybe leave in when the feature was made available, then clarify how to enable it.

Reply:
`**Configuration Change. ** This feature is available in Momentum 4.2 and later.
As of 4.3.0 users do not have to enable the feature only in 4.2`

Reply:
`As of 4.3.0 users do not have to enable the module, but the module must be enabled in 4.2.x` sounds a little better.